### PR TITLE
Feature/download data

### PR DESCRIPTION
**Pull request by kamiltomczykdv**
Download data and create list

### CODE REVIEW

File: app/src/main/java/com/example/mymovieapp/repository/Repository.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            .addConverterFactory(GsonConverterFactory.create())
+            .baseUrl(baseUrl)
+            .build()
+
+        userService = retrofit.create(UserService::class.java)
+        val call = userService.getResponse(ts, apiKey, hash, limit, offset, orderBy)
+
+        call.enqueue(object : Callback<Response> {
+            override fun onFailure(call: Call<Response>, t: Throwable) {
+
```
Comment (norberthanysz) [inline app/src/main/java/com/example/mymovieapp/repository/Repository.kt:43]: add some error message, or log if the call returns an error

File: app/src/main/java/com/example/mymovieapp/view/movieList/MovieAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        holder.id.text = viewModel.response.value?.data?.results?.get(position)?.id.toString()
+        Glide
+            .with(context)
+            .load(
+                viewModel.getUrl(
+                    viewModel.response.value?.data?.results?.get(position)?.thumbnail?.path,
+                    viewModel.response.value?.data?.results?.get(position)?.thumbnail?.extension
+                )
+            )
+            .override(200, 200)
```
Comment (norberthanysz) [inline app/src/main/java/com/example/mymovieapp/view/movieList/MovieAdapter.kt:40]: hardcode

File: app/src/main/res/layout/fragment_host.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recyclerView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_marginStart="4dp"
+        android:layout_marginLeft="4dp"
+        android:layout_marginTop="4dp"
+        android:layout_marginEnd="4dp"
```
Comment (norberthanysz) [inline app/src/main/res/layout/fragment_host.xml:17]: MarginEnd is enough, you don't need to add marginRight cuz it's duplicated

Comment (norberthanysz) [reply app/src/main/res/layout/fragment_host.xml:17]: the same MarginStart and MarginLeft

Comment (norberthanysz) [reply app/src/main/res/layout/fragment_host.xml:17]: and also delete hardcodes

File: app/src/main/res/layout/single_movie.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+    <LinearLayout
+        android:id="@+id/single_movie_linear_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <ImageView
+            android:id="@+id/thumbnail_imageView"
+            android:layout_width="100dp"
```
Comment (norberthanysz) [inline app/src/main/res/layout/single_movie.xml:16]: hardcodes, please look all your app and move all values to resources, strings, and colors XML

File: app/src/main/java/com/example/mymovieapp/MainActivity.kt [hunk 5]
```diff
@@ -2,10 +2,14 @@ package com.example.mymovieapp
...
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.navigation.fragment.NavHostFragment
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var viewModel: ViewModel
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        val host = NavHostFragment.create(R.navigation.nav_graph)
```
Comment (norberthanysz) [inline app/src/main/java/com/example/mymovieapp/MainActivity.kt:12]: refactor this as we discussed in the office, just delete these 2 lines 😄

**Review by norberthanysz — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/mymovieapp/model/Data.kt [hunk 6]
```diff
@@ -0,0 +1,9 @@
+package com.example.mymovieapp.model
+
+data class Data(
+    val offset:Int,
```
Comment (krzysztofDz) [inline app/src/main/java/com/example/mymovieapp/model/Data.kt:4]: Please use refactor tool here. There should be space after : sign ;) On mac this is option + command + L shortcut.

**Review by norberthanysz — APPROVED** [review]

**Pull request merged**